### ISSUE TITLE
Login issue - Session expired

### ISSUE THREAD

**Issue by sitemapxml** [authentic-theme/authentic-theme#1480]
When I try to log in to Webmin I got "Session expired" warning, multiple times in a row. I tried to delete cookies and cache but with no success. I am using Firefox 77.0.1 (64-bit) and leatest Webmin and Authentic theme version. I got this error only in Firefox. In Chrome it works just fine. 

![webmin](https://user-images.githubusercontent.com/33502827/85347979-72426900-b4fa-11ea-9e91-f5429e68281b.gif)

**Comment by iliarostovtsev** [comment]
Hi,

I cannot reproduce it in my Firefox 77. It's evident that something is destroying a session cookie. Try disabling all browser's extensions and try again and/or try incognito mode.

**Comment by sitemapxml** [comment]
Hi, 
I have disabled all browser plugins, deleted all website data and cache and message is still popping. From some reason (probably because of plugins) it was still showing in private mode, now it is working in private mode, but not in regular window.

The cookies I currently have are:
- redirect:1
- testing:1
- sid:8dd7d5ffc23c8345e063a1ed5193c143

I have activated cookie manager plugin after loading a page.
